packages/react-dom/src/client/ReactDOMComponent.js

```diff
@@ -1209,9 +1312,10 @@ export function warnForDeletedHydratableElement(
     }
     didWarnInvalidHydration = true;
     console.error(
-      'Did not expect server HTML to contain a <%s> in <%s>.',
-      child.nodeName.toLowerCase(),
-      parentNode.nodeName.toLowerCase(),
+      'The content rendered by the server and the client did not match ' +
```

Comment:
The explanation in detail sounds great, but I feel there are too many words that are the same across multiple messages until the very essence of this message which is "the server has rendered an extra text node". Could be easier and quicker to grasp the gist if the varying part of the message was at the beginning, and the repetitive explanation at the end.

Reply:
Yeah good point. I'll take another pass to clean them up after all of them are covered.